helm_deploy/values-dev.yaml

```diff
@@ -31,3 +31,6 @@ generic-prometheus-alerts:
 generic-data-analytics-extractor:
   serviceAccountName: hmpps-education-and-work-plan-dev-to-ap-s3
   cronJobSchedule: '00 7 * * 1-5' # Start at 7am UTC Monday-Friday to allow for the fact that we terminate the resources (inc. RDS) overnight and all weekend in the dev env (see scheduledDowntime above)
+
+hmpps-education-and-work-plan-api-queue-housekeeping:
+  schedule: "*/10 07-21 * * 1-5" # Every 10 minutes between 7am and 9pm UTC Monday-Friday to allow for the fact that we terminate the resources overnight and all weekend in the dev env (see scheduledDowntime above)
```

Comment:
The cron expression is correct; what I'm not sure about is whether I've referenced it correctly in the helm values.
I guess we will tell once it's been merged! 🤣